python_modules/libraries/dagster-dbt/dagster_dbt_tests/test_asset_decorator.py

```diff
@@ -34,6 +35,45 @@ def all_dbt_assets(context, dbt: DbtCli):
     ).success
 
 
+def test_json_manifest():
+    with open(manifest_path, "r") as f:
+        manifest_json = json.load(f)
```

Comment:
nit: use `pathlib` for opening the file
```suggestion
    with manifest_path.open() as f:
        manifest_json = json.load(f)
```